GliaWidgets/Component/CallButtonBar/Button/CallButton.swift

```diff
@@ -131,7 +131,9 @@ class CallButton: UIView {
             badgeItemCount: badgeView?.newItemCount
         )
         accessibilityValue = properties.value
-        accessibilityLabel = properties.label
+
+        let buttonAccessibilityLabel = "audio_\(properties.value.lowercased())_button"
+        accessibilityLabel = isUserInteractionEnabled ? buttonAccessibilityLabel : ""
```

Comment:
Can Appium find element by accessibilityLabel? 
Looks a bit strange you use `accessibilityLabel` instead of `accessibilityIdentifier`

Reply:
I'm not sure, I think this is just typo :)

Reply:
I definitely meant identifier, but the test worked with both... Fixed it now

Reply:
afaik, Appium when search item starts from identifier > label > if label or text item compares text.